metricflow/test/model/validations/test_identifiers.py

```diff
@@ -68,6 +69,7 @@ def test_invalid_composite_identifiers() -> None:  # noqa:D
                                 type=DimensionType.TIME,
                                 type_params=DimensionTypeParams(
                                     is_primary=True,
+                                    time_format="YYYY-MM-DD",
```

Comment:
I think these are a legacy holdover, and we've been removing them from the various models, so unless this is absolutely necessary for this PR to work correctly we should revert the time_format additions before merging.

Reply:
New commit removes these.